### ISSUE TITLE
tf.metrics.* should be exposed

### ISSUE THREAD

**Issue by caisq** [tensorflow/tfjs#208]
To get help from the community, check out our [Google group](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs).

#### TensorFlow.js version
0.9.1

**Comment by dsmilkov** [comment]
Losses are already exposed: https://github.com/tensorflow/tfjs-core/blob/master/src/ops/ops.ts#L225

I think the problem in the mailing list might be something else.

**Comment by caisq** [comment]
@dsmilkov Acknowledged, thanks for pointing this out.

**Comment by caisq** [comment]
It is a little misleading that the methods currently under tf.losses.* are from tfjs-core. Maybe we should include the loss functions from tfjs-layers in it as well. There are a few missing in tfjs-core, such as `categoricalHinge`.
